### DISCUSSION TITLE
请问各位Bilibili Evolved用户，【批量导出高清图片】功能如何使用？

### DISCUSSION THREAD

**Discussion by Meloetta-Nahida** [the1812/Bilibili-Evolved#4990] [提问 / Q&A]
请问各位Bilibili Evolved用户，【批量导出高清图片】功能如何使用？我在这个项目的Github介绍主页也没看见详细的介绍，Bilibili Evolved的功能比较多，我也不是很清楚这些功能到底应该如何使用？

**Comment by Geeyun-JY3** [comment]
[`图片批量导出`](https://github.com/the1812/Bilibili-Evolved/blob/dbd813b10041aeec0a4e1867be4a4f75c835bd5b/doc/features/features.md#%E5%9B%BE%E7%89%87%E6%89%B9%E9%87%8F%E5%AF%BC%E5%87%BA)？其专栏的入口在功能面板，动态的入口在各动态卡片右上角更多菜单里。
不过该功能最近在动态卡片失效了：#4976

仓库里有功能介绍文档：https://github.com/the1812/Bilibili-Evolved/blob/master/doc/features/features.md
在脚本设置面板中点击组件打开的二级面板也有各自的介绍，与上述文档是一致的。
